designer/client/test/VisualizationUrl-test.js

```diff
@@ -33,4 +33,15 @@ describe("validating url counts extractors", () => {
         };
         expect(extractCountParams(params).to.isSame(moment(sampleTimestamp))).toBe(true);
     });
+
+    it("extract duration from string", () => {
+        expect(extractCountParams({ to: "0", refresh: "1s" }).refreshIn).toBe(1);
+        expect(extractCountParams({ to: "0", refresh: "1m" }).refreshIn).toBe(60);
+        expect(extractCountParams({ to: "0", refresh: "1h" }).refreshIn).toBe(3600);
+
+        expect(extractCountParams({ to: "0" }).refreshIn).toBe(false);
+        expect(extractCountParams({ to: "0", refresh: "" }).refreshIn).toBe(false);
+        expect(extractCountParams({ to: "0", refresh: "1000" }).refreshIn).toBe(false);
+        expect(extractCountParams({ to: "0", refresh: "1d" }).refreshIn).toBe(false);
```

Comment:
"1d" is not a valid refresh value but "24h" (or more) is fine

Reply:
this param is just iso part after T. it doesn't have to be bulletproof. Durations like 1d, even 6h, make no sense and i don't think we should care such users. harmless, not dangerous, not worth attention